tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -416,3 +422,123 @@ def test_positive_e2e_crud(session, module_org, module_loc):
+
+    :id: cd19b90d-836f-4efd-ccbc-d5e09a909a67
+
+    :setup: User with reporting access rights, some host with applied errata
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Applied Errata -> Generate
+        3. Submit
+
+    :expectedresults: A report is generated with all applied errata listed
+
+    :CaseImportance: Medium
+    """
+
+
+@tier2
+@stubbed()
+def test_datetime_picker(session):
+    """ Generate an Applied Errata report with date filled
+
+    :id: cd19b90d-836f-4efd-c1bc-d5e09a909a67
+
+    :setup: User with reporting access rights, some host with applied errata at
+            time A, some host (can be the same host) with applied
+            errata at time B>A and no errata applied at other time
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Applied Errata -> Generate
+        3. Fill in timeFrom>A
+        4. Fill in B>timeTo
+        4. Generate
+
+    :expectedresults: A report is generated with all applied errata listed
+                      that were generated before timeFrom and timeTo
+
+    :CaseImportance: Medium
+    """
+
+
+@tier2
+@stubbed()
+def test_positive_autocomplete(session):
+    """ Check if host field suggests matching hosts on typing
+
+    :id: cd19b90d-836f-4efd-c2bc-d5e09a909a67
+
+    :setup: User with reporting access rights, some Host, some report with host input
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Registered Hosts -> Generate
+        3. Fill in part of the Host name
+        4. Check if the Host is within suggestions
+        5. Select the Host
+        6. Submit
+
+    :expectedresults: The same report is generated as if the host has been entered manually
+
+    :CaseImportance: Medium
+    """
+
+
+@tier3
+@stubbed()
+def test_positive_schedule_generation_and_get_mail(session):
+    """ Schedule generating a report. Request the result be sent via e-mail.
+
+    :id: cd19b90d-836f-4efd-c3bc-d5e09a909a67
+
+    :setup: User with reporting access rights, some Host
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Registered Hosts -> Generate
+        3. Set schedule to current time + 1 minute
+        4. Check that the result should be sent via e-mail
+        5. Submit
+        6. Receive the e-mail
+
+    :expectedresults: After ~1 minute, the same report is generated as if
+                      the results were downloaded from WebUI
+
+    :CaseImportance: Medium
+    """
+
+
+@tier2
+@stubbed()
+def test_negative_bad_email(session):
+    """ Generate a report and request the result be sent to
+        a wrong formatted e-mail address
+
+    :id: cd19b90d-836f-4efd-c4bc-d5e09a909a67
+
+    :setup: User with reporting access rights, some Host, some report with host input
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Registered Hosts -> Generate
+        3. Check that the result shoudl be sent via e-mail
+        4. Submit
+        5. Receive the e-mail
```

Comment:
if the email address is wrongly formatted, I guess it should be sent via email right? instead, user should see the error